spec/lib/traject/config/format_main_spec.rb

```diff
@@ -1348,144 +1292,94 @@
+          end
+        end
 
-  context 'Leader/06 = t and Leader/07 = c and 999m = LANE-MED' do
-    let(:record) do
-      MARC::Record.new.tap do |r|
-        r.leader = '01952ctc  2200457Ia 4500'
-        r.append(MARC::DataField.new('999', ' ', ' ',
-                                     MARC::Subfield.new('m', 'LANE-MED')))
+        it 'is a Book and not Archive/Manuscript' do
+          expect(result[field]).to eq ['Book']
+        end
       end
     end
 
-    it 'is a Book and not Archive/Manuscript' do
-      expect(result[field]).to eq ['Book']
-    end
-  end
+    context 'when leader/06 is t' do
+      context 'when leader/07 is d' do
+        let(:record) do
+          MARC::Record.new.tap do |r|
+            r.leader = '01952ctd  2200457Ia 4500'
+            r.append(MARC::DataField.new('999', ' ', ' ',
+                                        MARC::Subfield.new('m', 'LANE-MED')))
+          end
+        end
 
-  context 'Leader/06 = a and Leader/07 = c and 999m not LANE-MED' do
-    let(:record) do
-      MARC::Record.new.tap do |r|
-        r.leader = '01952cac  2200457Ia 4500'
-        r.append(MARC::DataField.new('999', ' ', ' ',
-                                     MARC::Subfield.new('m', 'GREEN')))
+        it 'is a Book and not Archive/Manuscript' do
+          expect(result[field]).to eq ['Book']
+        end
       end
-    end
 
-    it 'is a Archive/Manuscript' do
-      expect(result[field]).to eq ['Archive/Manuscript']
-    end
-  end
+      context 'when leader/07 is c' do
+        let(:record) do
+          MARC::Record.new.tap do |r|
+            r.leader = '01952ctc  2200457Ia 4500'
+            r.append(MARC::DataField.new('999', ' ', ' ',
+                                        MARC::Subfield.new('m', 'LANE-MED')))
+          end
+        end
 
-  context 'Leader/06 = t and Leader/07 = c and 999m not LANE-MED' do
-    let(:record) do
-      MARC::Record.new.tap do |r|
-        r.leader = '01952cac  2200457Ia 4500'
-        r.append(MARC::DataField.new('999', ' ', ' ',
-                                     MARC::Subfield.new('m', 'GREEN')))
+        it 'is a Book and not Archive/Manuscript' do
+          expect(result[field]).to eq ['Book']
+        end
       end
     end
-
-    it 'is a Archive/Manuscript' do
-      expect(result[field]).to eq ['Archive/Manuscript']
-    end
   end
 
-  context 'Leader/06 not a and not t and Leader/07 = c and 999m = LANE-MED' do
-    let(:record) do
-      MARC::Record.new.tap do |r|
-        r.leader = '01952c c  2200457Ia 4500'
-        r.append(MARC::DataField.new('999', ' ', ' ',
-                                     MARC::Subfield.new('m', 'LANE-MED')))
+  context 'when leader/06 is a' do
+    context 'when leader/07 is c' do
+      let(:record) do
+        MARC::Record.new.tap do |r|
+          r.leader = '01952cac  2200457Ia 4500'
+        end
       end
-    end
 
-    it 'is has no format' do
-      expect(result[field]).to eq nil
```

Comment:
These no format tests were not actually testing anything in our code, so they've been discarded.